tests/test_tensorflow_sparse_data_api.py

```diff
@@ -23,122 +21,146 @@
 ROWS = 1000
 
 
-@pytest.fixture(
-    params=[
-        {
-            "input_shape": (10,),
-        },
-    ]
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        (10,),
+    ],
+)
+# We test for single and multiple attributes
+@pytest.mark.parametrize(
+    "num_of_attributes",
+    [1, 2, 3],
 )
-def model(request):
-    model = Sequential()
-    model.add(tf.keras.Input(shape=request.param["input_shape"], sparse=True))
-    # TODO: TF https://github.com/tensorflow/tensorflow/issues/47532
```

Comment:
Are these issues resolved? Let's bookkeep them if we don't to be present in code.

Reply:
I don't think so. I will add them in the TF Sparse reader implementation.